microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -94,24 +203,48 @@ https://login.microsoftonline.com/5c9eedce-81bc-42f3-8823-48ba6258b391/oauth2/v2
 }
 ```
 
-For more information, see [Get a token](/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow#get-a-token) in _Microsoft identity platform and the OAuth 2.0 client credentials flow_.
+For more information, see [Get a token](/azure/active-directory/develop/v1.1-oauth2-client-creds-grant-flow#get-a-token) in _Microsoft identity platform and the OAuth 2.0 client credentials flow_.
+
+---
 
 
 <!-- ====================================================================== -->
 ## Using the API endpoints
 
-After you have an access token, you can use the Microsoft Edge Add-ons API.  This API exposes endpoints for getting a list of products, updating products, and publishing products.
+The Update REST API exposes endpoints for getting a list of products, updating products, and publishing products.
 
-> [!NOTE]
-> There's no API for creating a new product or updating a product's metadata. For example, the description.  You must complete these tasks manually in Microsoft Partner Center.
+##### [v1.1](#tab/v1-1)
+
+After you have an API key, you can use the Microsoft Edge Add-ons API.  
+
+##### [v1](#tab/v1)
+
+After you have an access token, you can use the Microsoft Edge Add-ons API.  
+
+---
 
-The API is available at the endpoint `https://api.addons.microsoftedge.microsoft.com`
+There's no API for creating a new product or updating a product's metadata, such as the description.  You must complete these tasks manually in Microsoft Partner Center.
+
+The API is available at the endpoint `https://api.addons.microsoftedge.microsoft.com`.
 
 
 <!-- ====================================================================== -->
 ## Uploading a package to update an existing submission
 
-Use this API to update the package for an add-on.  This API uploads a package to update an existing draft submission of an add-on product.
+Use this REST API endpoint to update the package for an add-on.  This API uploads a package to update an existing draft submission of an add-on product.
+
+##### [v1.1](#tab/v1-1)
+
+<!-- todo: update for 1.1 -->
```

Comment:
update 8 code blocks  from v1 to v1.1, in "Using" article > "Sample request" section (4x) & bigger section above it (4x).  Not sure if simple copypaste from equiv Ref article section.
@yvsubhash @bhuvanapriyap 

Rendered parent section 1 of 4:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-3269&tabs=v1#uploading-a-package-to-update-an-existing-submission

Reply:
v1.1 code in tabs is ready for review.  closing.